ChegeBryan/politico
# #163684121 Edit party

#### What does this PR do?

Builds the endpoint to edit a party detail

#### Description of Task to be completed?

- [x] Write a unit test for the edit endpoint 
- [x] Build out necessary functions to perform the edit 

#### How should this be manually tested?
After cloning the repository to your local machine - check [README.md](README.md) for instructions checkout to `ft-edit-party-163684121` branch. [Install the requirements](https://github.com/ChegeBryan/politico#installing) and launch the flask server check [here](https://github.com/ChegeBryan/politico#starting-the-server) for instructions. Using [Postman](https://www.getpostman.com/) open create a `POST` request with this header
`Key Content-Type: application/json`.
Put this in the `json` body
```JSON
{
	"party_name": "Actionated Party",
	"hq_address": "Mountain View 23"
}
```
Change the details you want for the party by making a `PUT` request e.g 
```JSON
{
	"party_name": "Actionated",
	"hq_address": "Mountain 23"
}
```

#### What are the relevant pivotal tracker stories?
[#163684121](https://www.pivotaltracker.com/story/show/163684121)

#### Postman Documentation.
[Postman Docs](https://documenter.getpostman.com/view/4869820/RztoM8mV)

#### Screenshots.
##### `PUT /parties/<party_id>/edit`
![screenshot from 2019-02-07 07-52-53](https://user-images.githubusercontent.com/40359451/52391406-09498500-2aae-11e9-94a8-58f7cb3f1a90.png)
